components/DetectShake.js

```diff
@@ -0,0 +1,34 @@
+import { Accelerometer } from 'expo-sensors'
+
+// Threshold sets shake sensitivity - A lower number gives higher ensitivity. A higher number gives lower sensitivity.
+const threshold = 190
+
+export class DetectShake {
```

Comment:
Nice usage of outer code ;) I encourage you to deeply investigate how classes work in JavaScript!